sdk/python/lib/pulumi/resource.py

```diff
@@ -26,6 +26,8 @@
 
 from .metadata import get_project, get_stack
 
+from . import _types
```

Comment:
Why all the line breaks in the imports here?

Reply:
I added a commit that cleans this up.